variables.tf

```diff
@@ -14,6 +14,16 @@
  * limitations under the License.
  */
 
+variable "create_command" {
+  description = "On create, the command you'd like to run with the GCP SDK."
+  default     = "true"
```

Comment:
Is `true` really a sensible command to run?

Reply:
Was looking for a no-op default when someone doesn't supply a create or destroy command here, and `true` seems to fit that for me. Do you have a different suggestion?

Reply:
Actually nevermind, we'll need to switch this up anyway with the suggestion below.